web_service/urls.py

```diff
@@ -14,8 +14,14 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('file_upload.urls')),
 ]
+
+if settings.DEBUG:
```

Comment:
Do u want to give access to the file only in debug mode? or u also want to open that for production also? I think this option should be open for production also.